### ISSUE TITLE
Github Repository Image not parsed properly

### ISSUE THREAD

**Issue by MuhtasimTanmoy** [LeonardoCardoso/SwiftLinkPreview#144]
**Using:**
```
SwiftLinkPreview 3.3.0
Xcode 12.4
```

**Issue:** 
```
Invalid response image attribute in case of any GitHub URL.
```

**Reproduce By:**
```
Request URL:

  https://github.com/LeonardoCardoso/SwiftLinkPreview

Response Wrong Attribute:

  image: Optional("https://github.githubassets.com/images/search-key-slash.svg")

Should be:

  image: Optional("https://opengraph.githubassets.com/ff9150cfc28a8413bb2fc25c4b82ebf0539
				 7aa3e51e0642527dc68aabdeb1368/LeonardoCardoso/SwiftLinkPreview")

```

**Reason:**

	Failing to parse the content of this tag <meta property="og:image" content=""> for opengraph image.

**Comment by LeonardoCardoso** [comment]
Thanks for spotting this. I don't have the capacity to work on this right now. Feel free to contribute and open a PR.

**Comment by MuhtasimTanmoy** [comment]
Kindly check the pull request. Working for all Github links, and previously tested links.

**Comment by LeonardoCardoso** [comment]
@MuhtasimTanmoy thanks! I will check ASAP.

**Comment by MuhtasimTanmoy** [comment]
Hi, just following up regarding this issue.

**Comment by LeonardoCardoso** [comment]
Fixed in version 3.4.0
